src/Ormolu/Terminal/Monad.hs

```diff
@@ -0,0 +1,7 @@
+module Ormolu.Terminal.Monad ((>>)) where
```

Comment:
Nitpick/feel free to ignore: "Monad" feels slightly misleading as `Term` is no `Monad`; could be `Ormolu.Terminal.Monad` (for comparison: `template-haskell` calls their `QualifiedDo` module [`Language.Haskell.TH.CodeDo`](https://hackage.haskell.org/package/template-haskell-2.19.0.0/docs/Language-Haskell-TH-CodeDo.html)).